documentation/getstarted/firststeps/commandline.md

```diff
@@ -69,17 +84,23 @@ and execute it by issuing:
 
 You now get:
 
-<pre class="console">Database: jdbc:h2:file:./foobardb (H2 1.4)
+<pre class="console">Database: jdbc:postgresql://instances.spawn.cc:port/foobardb (PostgreSQL 11.0)
 Successfully validated 2 migrations (execution time 00:00.018s)
 Current version of schema "PUBLIC": 1
 Migrating schema "PUBLIC" to version 2 - Add people
 Successfully applied 1 migration to schema "PUBLIC" (execution time 00:00.016s)</pre>
 
+Any time you want to reset your database back to it's original state, in this case an empty instance, you can run a reset command on your Spawn data container:
+
+<pre class="console"><span>&gt;</span> spawnctl reset data-container flyway-container</pre>
+
+You can then re-run migrate on your database.
+
```

Comment:
This is the spawn-specific way of doing this and probably won't apply to many users in the long-run if the intention is they only use spawn during this getting started section as opposed to permanently using spawn

Could it also mention using `flyway clean` which achieves the same thing? Or perhaps lines 93-98 aren't required to get started?